test/test-client.js

```diff
@@ -1,112 +1,58 @@
 var assert = require('assert');
 var ImgixClient = require('../src/imgix-core-js');
-var sinon = require('sinon');
```

Comment:
Is there any reason not to just leave sinon in here for use in the future? It's listed as a dev dependency, so keeping it around won't end up affecting any end-users.

Reply:
Good point, I did it for cleanliness but I'm not opposed to keeping it - especially if it's needed again in the future.